### PR TITLE
PART 1 - Features/application submit sync #108846424 

### PR DESCRIPTION
**Pull request by chrisdolendo**
This is Part 1 of the application submit sync. It involves updating the application model with new terminology that fits better with Salesforce's terminology. There is still some massaging left to do but not as much now.

### CODE REVIEW

File: app/assets/javascripts/short-form/templates/b2-contact.html.slim [hunk 1]
```diff
@@ -39,16 +39,16 @@ form novalidate="" name="form.applicationForm" ng-submit="submitForm({callback:
...
-      small.error ng-if="inputInvalid('second_phone_number')"
+        input#applicant_second_phone(ng-model="applicant.alternatePhone" name="alternatePhone" type="tel" ng-required="applicant.additionalPhone" ng-class="{ error: inputInvalid('alternatePhone') }" ng-pattern="/[^a-zA-Z]/" ng-change="applicant.additionalPhone = true" ui-mask="(999) 999-9999" placeholder="(555) 555-5555" ui-mask-placeholder)
+      small.error ng-if="inputInvalid('alternatePhone')"
         | {{'ERROR.PHONE_NUMBER' | translate}}
-      label.sr-only for="applicant_second_phone_number_type" translate="LABEL.WHAT_TYPE_OF_NUMBER"
-      select#applicant_second_phone_number_type ng-model="applicant.second_phone_number_type" data-width="auto" name="second_phone_number_type" ng-required="applicant.additionalPhone" ng-class="{ error: inputInvalid('second_phone_number_type') }"
+      label.sr-only for="applicant_second_phone_type" translate="LABEL.WHAT_TYPE_OF_NUMBER"
+      select#applicant_second_phone_type ng-model="applicant.alternatePhoneType" data-width="auto" name="alternatePhoneType" ng-required="applicant.additionalPhone" ng-class="{ error: inputInvalid('alternatePhoneType') }"
```
Comment (menslow) [inline app/assets/javascripts/short-form/templates/b2-contact.html.slim]: Might make sense to change this to `select#alternatePhoneType`

File: app/assets/javascripts/short-form/templates/b2-contact.html.slim [hunk 2]
```diff
@@ -39,16 +39,16 @@ form novalidate="" name="form.applicationForm" ng-submit="submitForm({callback:
...
-      small.error ng-if="inputInvalid('second_phone_number')"
+        input#applicant_second_phone(ng-model="applicant.alternatePhone" name="alternatePhone" type="tel" ng-required="applicant.additionalPhone" ng-class="{ error: inputInvalid('alternatePhone') }" ng-pattern="/[^a-zA-Z]/" ng-change="applicant.additionalPhone = true" ui-mask="(999) 999-9999" placeholder="(555) 555-5555" ui-mask-placeholder)
+      small.error ng-if="inputInvalid('alternatePhone')"
         | {{'ERROR.PHONE_NUMBER' | translate}}
-      label.sr-only for="applicant_second_phone_number_type" translate="LABEL.WHAT_TYPE_OF_NUMBER"
-      select#applicant_second_phone_number_type ng-model="applicant.second_phone_number_type" data-width="auto" name="second_phone_number_type" ng-required="applicant.additionalPhone" ng-class="{ error: inputInvalid('second_phone_number_type') }"
+      label.sr-only for="applicant_second_phone_type" translate="LABEL.WHAT_TYPE_OF_NUMBER"
```
Comment (menslow) [inline app/assets/javascripts/short-form/templates/b2-contact.html.slim]: Might make sense to change this to `for="alternatePhoneType"`

**Comment by menslow** [comment]
@chrisdolendo Overall things look great!

Couple things:
1. Just want to confirm that this is still not synching with Salesforce at this point, correct?
2. This would mean a lot of extra work, but does it make sense to make input “names" and label “for” attributes to align with the new field names?

**Comment by chrisdolendo** [comment]
@menslow not yet synching to Salesforce but Part 2 will include that code.

will create a chore for the names and labels!

**Pull request merged**